File: src/engine/ChangeLens.Core/Publication/Models/ReadingTrust.cs
namespace ChangeLens.Core.Publication.Models;

/// <summary>Describes the strongest trust level attached to a published claim.</summary>
public enum ReadingTrust
{
    /// <summary>No checker verdict supports the claim.</summary>
    Unchecked,

    /// <summary>The claim is derived from checked claims.</summary>
    Derived,

    /// <summary>The checker supports the claim.</summary>
    Checked,
}

File: src/desktop/src-tauri/src/analysis/models/reading_trust.rs
use serde::{Deserialize, Serialize};

/// Describes the verification trust level of a claim or citation provenance.
#[derive(Clone, Copy, Debug, Deserialize, PartialEq, Eq, Serialize)]
#[serde(rename_all = "camelCase")]
pub enum ReadingTrust {
    Unchecked,
    Derived,
    Checked,
}
